operators/src/processing/reprojection.rs

```diff
@@ -275,13 +276,22 @@ impl RasterOperator for Reprojection {
             tiling_spec: context.tiling_specification(),
         };
 
-        let initialized_operator = InitializedRasterReprojection {
+        let op = InitializedRasterReprojection {
             result_descriptor: out_desc,
             source: raster_operator,
             state,
         };
 
-        Ok(initialized_operator.boxed())
+        Ok(context.initialize_operator(op.boxed(), Box::new(ReprojectionSpan {})))
+    }
+}
+
+#[derive(Clone)]
+struct ReprojectionSpan {}
+
+impl CreateSpan for ReprojectionSpan {
+    fn create_span(&self) -> Span {
+        span!(Level::TRACE, "Reprojection")
```

Comment:
Maybe we should create a macro for that.

Reply:
if we decide that this is the way to go for span creation we should.